v2/printings/midnight_sun.json

```diff
@@ -1,4 +1,63 @@
 [
+  {
+    "card_id": "esa_afontov_eco_insurrectionist",
+    "card_set_id": "midnight_sun",
+    "flavor": "Waiting is useless. The crisis is here; pick a side.",
+    "id": "33001",
+    "illustrator": "Benjamin Giletti",
+    "position": 1,
+    "printed_is_unique": false,
+    "printed_text": "The first time each turn you suffer core damage, you may draw 1 card and sabotage 2. <em>(The Corp trashes 2 cards of their choice from HQ and/or the top of R&D.)</em>",
+    "quantity": 1,
+    "stripped_printed_text": "The first time each turn you suffer core damage, you may draw 1 card and sabotage 2. (The Corp trashes 2 cards of their choice from HQ and/or the top of R&D.)"
+  },
+  {
+    "card_id": "chastushka",
+    "card_set_id": "midnight_sun",
+    "flavor": "We're all alike down here Androids are friends we salute Give us all a f***ing break Ol' Jack is the s*** on my boot.",
```

Comment:
```suggestion
    "flavor": "We're all alike down here\nAndroids are friends we salute\nGive us all a f***ing break\nOl' Jack is the s*** on my boot.",
```

Reply:
yep, thanks.  I'll tidy up the sheet -> JSON converter to make sure this stays.  the sheet definitely has the newlines.